ci/run-k8s-e2e-tests.sh

```diff
@@ -31,8 +31,8 @@ RUN_NETWORK_POLICY=false
 RUN_E2E_FOCUS=""
 RUN_E2E_SKIP=""
 KUBECONFIG_OPTION=""
-DEFAULT_E2E_CONFORMANCE_FOCUS="\[Conformance\]"
-DEFAULT_E2E_CONFORMANCE_SKIP="\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[sig-cli\]|\[sig-storage\]|\[sig-auth\]|\[sig-api-machinery\]|\[sig-apps\]|\[sig-node\]|\[sig-instrumentation\]"
+DEFAULT_E2E_CONFORMANCE_FOCUS="\[sig-network\]|\[Conformance\]"
+DEFAULT_E2E_CONFORMANCE_SKIP="\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[GCE\]|\[Ingress\]|\[NetworkPolicy\]|\[IPv6\]|\[Conformance\]\[Feature:.+\]|\[Feature:IPv6DualStack\]|\[Feature:IPv6DualStackAlphaFeature\]|\[sig-cli\]|\[sig-storage\]|\[sig-auth\]|\[sig-api-machinery\]|\[sig-apps\]|\[sig-node\]|\[sig-instrumentation\]|should create pod that uses dns|should provide Internet connection for containers"
```

Comment:
There is a change in this line for changing `\[Feature:.+\]` to `\[Conformance\]\[Feature:.+\]`. Can you explain why we need this change? I thought if sig-network is run separately, we should skip "[Conformance]". If it's run with Conformance test, we should no skip "[Conformance]", right?

Reply:
Updated.